lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,12 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}/tools",
+        "cp #{project.name}.nuspec #{project.name}/",
+        "cp chocolateyInstall.ps1 chocolateyUninstall.ps1 #{project.name}/tools",
+        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '#{project.name}/tools' -xf -",
+        "(cd #{project.name}; #{self.drive_root}/ProgramData/chocolatey/bin/choco.exe pack)",
```

Comment:
Will the nupkg file be dropped into the directory where pack is being run? Or in the directory where the nuspec lives?